includes/discovery/sensors/current/ict-mps.inc.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * ict-mps.inc.php
+ *
+ * LibreNMS current sensor discovery module for ICT Modular Power System
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 Lorenzo Zafra
+ * @author     Lorenzo Zafra<zafra@ualberta.ca>
+ */
+
+// Output Current
+$outputCurrent = trim(snmp_get($device, 'outputCurrent.0', '-Oqv', 'ICTICT-MODULAR-POWER-SYSTEM-MIB'), '" ');
+if (!empty($outputCurrent)) {
+    $divisor = 1;
+    $index = 0;
+    $oid = '.1.3.6.1.4.1.39145.13.8.0';
+    $descr = 'Output Current (A)';
+    $type = 'ict-mps';
+    $currentValue = $outputCurrent / $divisor;
```

Comment:
You shouldn't need to apply the divisor before hand, that should be taken care of globally.

Any reason this isn't done as yaml?

Reply:
Removed unnecessary math. No particular reason it's not done in yaml.